mat/vector.go

```diff
@@ -219,6 +219,10 @@ func (v *VecDense) RawVector() blas64.Vector {
 	return v.mat
 }
 
+func (v *VecDense) SetRawVector(a blas64.Vector) {
```

Comment:
Doc comment? (also add for the already existing `RawVector` method).

Reply:
Done.